src/containers/main-page/preview/Certifications.tsx

```diff
@@ -0,0 +1,71 @@
+import { useIsFetching } from 'react-query';
+import { useEffect } from 'react';
+
+import {
+  Grid, Typography, Link,
+} from '@mui/material';
+import ArticleIcon from '@mui/icons-material/Article';
+
+import { useUserFromDb } from '../cv-form/components/fields/personal-information/lib/query-hooks';
+import theme from '../../../common/theme';
+import {
+  BoxWrapperStyled, CircleIconStyled, PaperWrapperStyled, SectionTitle,
+} from '../styled';
+import { CV_FORM_STEPS } from '../cv-form/utils/constants';
+
+const Certifications = function (): JSX.Element {
+  const { data, refetch } = useUserFromDb();
+  const isFetching = useIsFetching('db-user');
+
+  useEffect(() => {
+    if (isFetching) refetch();
+  }, [refetch, isFetching]);
+
+  return (
+    <PaperWrapperStyled
+      elevation={1}
+    >
+      <Grid container>
+        <BoxWrapperStyled>
+          <ArticleIcon color="primary" sx={{ margin: theme.spacing(1.75) }} />
```

Comment:
![image](https://user-images.githubusercontent.com/63092716/182612130-3ee49613-e90c-44d6-ba1f-bb9efde967fa.png)

Please change everywhere. Using extra dependencies is not good approach